src/channel.ts

```diff
@@ -147,12 +147,27 @@ export const SINGLE_DEF_CHANNELS: SingleDefChannel[] = flagKeys(SINGLE_DEF_CHANN
 // Using the following line leads to TypeError: Cannot read property 'elementTypes' of undefined
 // when running the schema generator
 // export type SingleDefChannel = typeof SINGLE_DEF_CHANNELS[0];
-export type SingleDefChannel = 'x' | 'y' | 'x2' | 'y2' |
-  'longitude' | 'latitude' | 'longitude2' | 'latitude2' |
-  'row' | 'column' |
-  'color' | 'fill' | 'stroke' |
-  'size' | 'shape' | 'opacity' |
-  'text' | 'tooltip' | 'href' | 'key';
+export type SingleDefChannel =
+  | 'x'
```

Comment:
This preceding `|` style is extremely terrible

Reply:
Is there an option in prettier?